react/utils/slug.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  SPACE_REPLACER,
+  SPEC_FILTER,
+  FILTER_TITLE_SEP,
+  MAP_CATEGORY_CHAR,
+  MAP_BRAND_CHAR,
+} from '../constants'
+
+const from =
+  'ÁÄÂÀÃÅČÇĆĎÉĚËÈÊẼĔȆÍÌÎÏŇÑÓÖÒÔÕØŘŔŠŤÚŮÜÙÛÝŸŽáäâàãåčçćďéěëèêẽĕȇíìîïňñóöòôõøðřŕšťúůüùûýÿžþÞĐđßÆa·/_,:;'
+const to =
+  'AAAAAACCCDEEEEEEEEIIIINNOOOOOORRSTUUUUUYYZaaaaaacccdeeeeeeeeiiiinnooooooorrstuuuuuyyzbBDdBAa------'
```

Comment:
shouldn't this be on the `constants` file?

Reply:
Not sure too. I just left it there to be the same way as it is at `search-graphql`.